app/views/browse/show.html.erb

```diff
@@ -31,6 +31,17 @@
   } %>
 <% end %>
 
+<% if page.recruitment_survey_url %>
```

Comment:
I forgot to add this to my original review, but I think you might need to include the view stylesheet at the top of this view:

```ruby
<% add_view_stylesheet("intervention") %>
```

Reply:
We don't use that stylesheet in this file because we don't want to change the `margin-top` for this page. We are using that only for the subtopic page.